coldfront/core/project/templates/project/project_detail.html

```diff
@@ -218,52 +218,51 @@ <h2 class="d-inline">
     <a name="users">Users</a>
     <span class="badge badge-secondary">{{project_users.count}}</span>
   </h2>
-    <div class="float-right">
-      {% if project.status.name != 'Archived' and is_allowed_to_update_project %}
-        {% comment %}
+      <div class="float-right">
+          {% if project.status.name != 'Archived' and is_allowed_to_update_project %}
+              {% comment %}
           <a class="btn btn-primary" href="{{mailto}}" role="button"><i class="far fa-envelope"></i> Email Project Users</a>
         {% endcomment %}
-        <a class="btn btn-success" href="{% url 'project-add-users-search' project.pk %}" role="button"><i class="fas fa-user-plus" aria-hidden="true"></i> Add Users</a>
-        <a class="btn btn-danger" href="{% url 'project-remove-users' project.pk %}" role="button"><i class="fas fa-user-times" aria-hidden="true"></i> Remove Users</a>
-      {% endif %}
-    </div>
-    <div class="float-right">
-      {% if project.status.name != 'Archived' and can_leave_project %}
-        <div class="modal fade" id="denial{{ user.pk }}Modal" tabindex="-1" role="dialog" aria-labelledby="denial{{ user.pk }}ModalLabel" aria-hidden="true">
-          <div class="modal-dialog" role="document">
-            <form action="{% url 'project-remove-self' project.pk %}" method="post">
-              {% csrf_token %}
-
-              <div class="modal-content">
-                <div class="modal-header">
-                  <p class="modal-title">Leave Project: {{ project.name }}</p>
-                  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                    <span aria-hidden="true">&times;</span>
-                  </button>
-                </div>
-
-                <div class="modal-body">
-                  <label>Are you sure? You will immediately be disallowed from submitting jobs under this project.</label>
-
-                  <br><b>User:</b> {{ user.email }}
-                  <br><b>Project:</b> {{ project.name }}
-                </div>
-
-                <div class="modal-footer">
-                  <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-                  <button class="btn btn-danger mr-1" type="submit">
-                    Yes, leave this project.
-                  </button>
-                </div>
+              <a class="btn btn-success" href="{% url 'project-add-users-search' project.pk %}" role="button"><i class="fas fa-user-plus" aria-hidden="true"></i> Add Users</a>
```

Comment:
Minor question: Is your editor somehow converting spaces to tabs and modifying the indent level for HTML?

It makes the diffs bigger than they actually are.

Reply:
Yea, I think its weird with html formatting. I'm using pycharm and don't think I changed any settings, but ill look into it.

Reply:
I've resolved the over-indents--let me know if the diff reflects what you changed.

In PyCharm, for an HTML file, in the bottom-right, there should be "K spaces" --> "Configure Indents for HTML...". Does yours differ from this?

![pycharm_settings](https://user-images.githubusercontent.com/10734545/150193380-6f70af0a-9248-41fd-a701-42e198c5d0ae.png)

Reply:
It was different in my editor. I changed it to match yours.